setup-env

```diff
@@ -102,23 +162,39 @@ fi
 
 set +o nounset
```

Comment:
Is the `nounset` flipping logic still needed after your changes here?

Reply:
@mcdonnnj I addressed this here [c8f0b1b](https://github.com/cisagov/skeleton-generic/pull/166/commits/c8f0b1b996958a5fa965343fa7e6295b7160cbf4)